### ISSUE TITLE
Flac download

### ISSUE THREAD

**Issue by Sildenafil99** [Bockiii/deezloadermx-docker#22]
Hi
I've installed your repo on my xpenology but even if I have selected the flac format as the download quality, I always download the mp3 version.
Obviously the same song with Deezloader 4.4.1 installed on the pc is downloaded in flac version.
Is a version problem? 
do you plan to upgrade to the 4.4.1?
Thx

**Comment by Bockiii** [comment]
Hi,

Deezloader RMX latest official version is 4.4.0.

https://notabug.org/RemixDevs/DeezloaderRemix

Anything else might be fixed by someone else, but Deezloader RMX is officially no longer under development.

You should switch over to Deemix, the official successor: https://gitlab.com/Bockiii/deemix-docker

The docker container basically runs the same, it has some QoL features (like predefined ARL for all users etc.).

I will not change the source for Deezloader RMX to a fixed source as that wouldn't be the official source and whoever updates it could write all kinds of BS into the code.

tl;dr: Deezloader ded, use deemix.
